pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/model/patch/DefaultDBElementBuilderTest.java

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.dsm.model.patch;
+
+import org.broadinstitute.dsm.TestHelper;
+import org.broadinstitute.dsm.db.structure.DBElement;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DefaultDBElementBuilderTest {
+
+    @BeforeClass
+    public static void initializeDB() {
+        TestHelper.setupDB();
```

Comment:
is it really necessary? I don't see any test which requires any DB connection

Reply:
@gmakharat Thank you for noticing it, it's redundant, just removing it.